.gitattributes

```diff
@@ -0,0 +1 @@
+* text=auto eol=lf
```

Comment:
I had to add this to fix eslint errors due to Windows changing line endings. More info here: https://stackoverflow.com/questions/9976986/force-lf-eol-in-git-repo-and-working-copy